data/simulations/FAH_SARS-CoV-2_spike.yml

```diff
@@ -0,0 +1,40 @@
+type: md
+title: Trajectories of the SARS-CoV-2 spike protein from Folding@Home
+description: >
+  All-atom MD simulations of the SARS-CoV-2 spike protein, simulated using Folding@Home. 
+  Simulations were run using either GROMACS or OPENMM and are stored as compressed binary XTC files. 
+  The dataset comprises 3 projects, each having a RUN*/CLONE*/result* directory structure. 
+  Each RUN represents a unique starting conformation, each CLONE is a unique MD run from the 
+  specified starting conformation, and each result is a fragment of the contiguous simulation.
+  
+  *Entire dataset:* To get the raw trajectory files (XTCs) of the whole dataset (7 TB), you can use the [AWS CLI](https://aws.amazon.com/cli/):
```

Comment:
Can you suggest a way to get just the longest trajectory, or a set of 100 representative trajectories, which would be much more useful and much smaller than the whole dataset? Or are these XTC files not knitted together yet?

Reply:
They're not knitted together. I'll add my FAST simulations, each of which are reasonably long